pyccl/tracers.py

```diff
@@ -518,8 +678,17 @@ def __init__(self, cosmo, dndz, has_shear=True, ia_bias=None,
                 a = tmp_a
             # Reverse order for increasing a
             t_a = (1./(1+z_a[::-1]), a[::-1])
-            self.add_tracer(cosmo, kernel=kernel_i, transfer_a=t_a,
-                            der_bessel=-1, der_angles=2)
+            if (cosmo['sigma_0'] == 0):
+                # GR case
+                self.add_tracer(cosmo, kernel=kernel_i, transfer_a=t_a,
+                                der_bessel=-1, der_angles=2)
+            else:
+                # MG case
+                k = np.logspace(-5, -0.7)
```

Comment:
Can not use the k sampling based on ells because these are specified and used at the level of the Cls that comes after the tracers are defined. However, the above is not an issue because this range is chosen to be good enough for   the interpolation of the MG parameters and also does not go beyond the linear regime.

Reply:
See other comments.